slack_bolt/context/base_context.py

```diff
@@ -91,6 +91,11 @@ def authorize_result(self) -> Optional[AuthorizeResult]:
         """The authorize result resolved for this request."""
         return self.get("authorize_result")
 
+    @property
+    def bot_access_token(self) -> Optional[str]:
```

Comment:
Decided to keep this because the original bot-token is still accessible from the function but hidden

Reply:
@WilliamBergamin can you make the name consistent with bolt-js? https://github.com/slackapi/bolt-js/pull/1567#discussion_r959075081

Reply:
The prop name is not yet updated -- we should use slack_function_bot_access_token instead.

Reply:
@seratch @srajiang I made the change to `slack_function_bot_access_token` but It feels long, I was thinking of changing it to `function_bot_access_token` but it would not be consistent with bolt-js

Reply:
@WilliamBergamin I know, it does feel long. I guess if you need a decider reasoning, I'd opt for consistency and clarity over brevity in this one case - in future when need to propagate changes across SDKs, it'll make finding things a little easier. SlackFunctionBotAccessToken in JS and slack_function_bot_access_token in Python are very unmistakeable.

Reply:
Ah, but on looking at your `Function` class, I see you've already opted to go with just `Function` over `SlackFunction`, so I do see how you could opt to be internally consistent here and omit `slack`.

Reply:
@srajiang I was also considering renaming `Function` to `SlackFunction` and `app.function` to `app.slack_function` but it felt long once I made the changes, now is a good time to make a more final decision on what should be implemented

Reply:
I rename to `slack_function` `function` can really become ambiguous and this is how it is done in bolt-js

Reply:
So - `app.function('id', new SlackFunction( ...))` is what we support at the top level, for brevity reasons only if that's helpful. We went with `SlackFunction` for the class because `new Function` just seemed wrong from JS perspective (everything is a function in this language + there's a reserved word so that makes lookup possibly a little more annoying for devs).

`SlackFunction` is what Deno Slack developers will instantiate as well for their handlers, but again we're all in the same JS boat together so that's probably why we've ended up going with the same naming here. 

I think at the end of the day, if `Function` can be considered by a Python dev to be clear and intuitive as to what they're creating, then we're okay. And of course we will always have to have our own versions of docs, so I defer to you on both where to strike the balance with this!

Reply:
Ahh I see, naming the class `SlackFunction` and using `app.function` should result in a clean and intuitive python user experience 🥇